docs/module_guides/models/embeddings.md

```diff
@@ -188,7 +188,9 @@ embeddings = embed_model.get_text_embedding(
 )
 ```
 
-## Modules
+(list_of_embeddings)=
+
```

Comment:
We use the Modules terminology pretty consistently in the docs -- we should probably try to stick to one way of saying this?